pypeit/spectrographs/p200_tspec.py

```diff
@@ -0,0 +1,269 @@
+""" Module for Keck/NIRES specific codes
+"""
+import numpy as np
+from astropy.time import Time
+
+from pypeit import msgs
+from pypeit import telescopes
+from pypeit.core import framematch
+from pypeit.par import pypeitpar
+from pypeit.spectrographs import spectrograph
+from pypeit.images import detector_container
+
+from pkg_resources import resource_filename
+
+
+class P200TSPECSpectrograph(spectrograph.Spectrograph):
+    """
+    Child to handle Keck/NIRES specific code
+    """
+    ndet = 1
+
+    def __init__(self):
+        # Get it started
+        super(P200TSPECSpectrograph, self).__init__()
+        self.spectrograph = 'p200_tspec'
+        self.telescope = telescopes.KeckTelescopePar()
```

Comment:
I don't know if `self.telescope` is used for much, but #990 added a `P200TelescopePar` for what it's worth

Reply:
Good catch. Thanks for that.